icu4c/source/common/unicode/unistr.h

```diff
@@ -327,6 +328,53 @@ class U_COMMON_API UnicodeString : public Replaceable
    */
   inline bool operator== (const UnicodeString& text) const;
 
+#ifndef U_HIDE_DRAFT_API
+  /**
+   * Equality operator. Performs only bitwise comparison with a
+   * std::u16string_view or (if U_SIZEOF_WCHAR_T==2) std::wstring_view.
+   * TODO: Should we document this as what it is, "convertible to one of these string views",
+   * rather than claiming that it needs to *be* one?
```

Comment:
WDYT?

Reply:
@richgillam one of the input types that is convertible is a raw pointer. The input can also be a std::u16string, or, with this PR, a UnicodeString. That's why it seems a little misleading to just document the input as a string view (and to call the template type StringView).

Reply:
rephrasing